documentation/en/chapter_1.md

```diff
@@ -0,0 +1,81 @@
+## 1. Motivation
+
+Even though telecom tecnology has taken a big leap in terms of development, it becomes evident it's own susceptibility. We're not talking about tecnology by itself but about people, those are the ones vulnerable to the side effects of communication isolation resulting from centralization, and regulation of communication standards worldwide.
+
+The advances that have emerged since the appearance of wireless networks in 1970, it's growth and demand in the last decade has been exponential. Specifically, there are two perspectives in this subject attending the underlying architecture of the network: wireless network infrastructure and infrastructurless wireless networks.
```

Comment:
```suggestion
The advances that have emerged since the appearance of wireless networks in 1970, it's growth and demand in the last decade has been exponential. Specifically, there are two perspectives in this subject attending the underlying architecture of the network: wireless network infrastructure and wireless networks infrastructurless .
```

Reply:
Network is the subject, wireless and infrastructure or infrastructureless are adjectives so I think the correct is the first one